config.yml

```diff
@@ -32,46 +34,50 @@ trading_options:
     - DOWN
     - UP
 
-  # Maximum number of trade 'slots' at any time (your USDT balance must be at least MAX_COINS * QUANTITY)
-  MAX_COINS: 7
+  # Maximum number of trade 'slots' at any time (your USDT balance must be at least TRADE_SLOTS * QUANTITY)
+  TRADE_SLOTS: 999
```

Comment:
Do not override default values in the config.  Please set back to default values for all

Reply:
have reset config values and have a separate config file now, thank you